angle_emb/train_cli.py

```diff
@@ -200,6 +216,7 @@ def main():
         epochs=args.epochs,
         learning_rate=args.learning_rate,
         save_steps=args.save_steps,
+        max_steps=args.max_steps,
```

Comment:
i've updated this implementation. `max_steps` can be passed to the trainer by argument_kwargs['max_steps'] = args.max_steps